stage1/iottymux/iottymux.go

```diff
@@ -410,9 +410,17 @@ func actionIOMux(statusFile string) error {
 	switch logMode {
 	case "k8s-plain":
 		var err error
-		// TODO(lucab): check what should be the target path with Euan
-		logTarget := filepath.Join(pathPrefix, appName, "logfile")
-		logFile, err = os.OpenFile(logTarget, os.O_CREATE|os.O_WRONLY|os.O_TRUNC, os.ModePerm)
+		logPath := os.Getenv("KUBERNETES_LOG_PATH")
+		logFullPath := filepath.Clean(filepath.Join("/rkt/kubernetes", logPath))
+
+		match, err := filepath.Match("/rkt/kubernetes/*", logFullPath)
+		if err != nil {
+			return fmt.Errorf("Couldn't analyze the full log path %s: %s", logFullPath, err)
+		} else if !match {
+			return fmt.Errorf("The log path isn't inside /rkt/kubernetes, which most probably means that user tried to escape that directory by using '..'")
+		}
+
+		logFile, err = os.OpenFile(logFullPath, os.O_CREATE|os.O_WRONLY|os.O_TRUNC, os.ModePerm)
```

Comment:
It seems that according to the CRI `KUBERNETES_LOG_PATH` can be a real path and not just a file name so this would fail if it turns out to be more than a file name.

Reply:
We can just add a comment mentioning this because I don't think it's an issue for now.

Reply:
Hmm... I don't think so, `logPath` is always a file name. I didn't see any code in kubelet or fragment of spec which proves otherwise. The name is just confusing. But yes, I can add a comment.